modules/dnf/module.yml

```diff
@@ -0,0 +1,25 @@
+name: dnf
+shortdesc: The dnf module offers pseudo-declarative package and repository management using dnf.
+example: |
+  type: dnf
+  repos:
+    - copr: atim/starship
```

Comment:
I think this can get confusing for users. This will be seen as an object and not a string. We could have a separate property `copr` that will take an array of strings which we could then iterate over and feed into the `dnf copr enable` command.

```yaml
type: dnf
copr:
  - atim/starship
  - ryanabx/cosmic-epoch
install:
  - starship
  - cosmic-desktop
```

Reply:
It looks less clean & more separated if it's done this way, hence why I went with `copr usr/project` format.

Is `:` making it an object or the whitespace?

Reply:
I can maybe have `COPR user/project` as format instead

Reply:
I changed it to `COPR user/project` format

Reply:
I still don't think this is better. Having a `copr:` property and a list of COPR repos would be very clear.